NuGet.Config

```diff
@@ -8,10 +8,15 @@
     <add key="myget-legacy@Local" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy%40Local/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
     <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption/nuget/v3/index.json" />
+    <add key="vs-impl" value="https://devdiv.pkgs.visualstudio.com/_packaging/vs-impl/nuget/v3/index.json" />
   </packageSources>
   <packageSourceMapping>
     <clear />
-    <packageSource key = "dotnet-public">
+    <packageSource key="vs-impl">
```

Comment:
We can't have private feeds in our OSS repo. I believe this feed is an upstream for msft_consumption (`vside`).

Reply:
yup! Working on getting the package published to the public vs-impl feed